.ci/safe-bash-scripts.md

```diff
@@ -0,0 +1,219 @@
+# Safe bash script configuration instructions
```

Comment:
Note sure if you want to keep it or not. I find it useful to have but I'm also fine with omitting it